python/apigw-lambda-rds/README.md

```diff
@@ -0,0 +1,44 @@
+# Api Gateway with Lambda & RDS (Mysql)
+
+This example creates an API Gateway with a Lambda that will query data from the RDS Mysql Instance. 
+
+After the RDS Instance is provisioned a Cloudformation custom resource is used to create a DB populate with some example data.
```

Comment:
reword the "create a DB populate" part to indicate whether the custom resource creating it, or populating it, or both.